runtime/src/block_cost_limits.rs

```diff
@@ -42,6 +42,9 @@ lazy_static! {
         (secp256k1_program::id(), COMPUTE_UNIT_TO_US_RATIO * 24),
         (ed25519_program::id(), COMPUTE_UNIT_TO_US_RATIO * 24),
         (system_program::id(), COMPUTE_UNIT_TO_US_RATIO * 5),
+        (compute_budget::id(), COMPUTE_UNIT_TO_US_RATIO * 5),
+        (solana_address_lookup_table_program::id(), COMPUTE_UNIT_TO_US_RATIO * 25),
+        (bpf_loader_upgradeable::id(), COMPUTE_UNIT_TO_US_RATIO * 775),
```

Comment:
It seems like these programs could have significantly different costs based on the instruction you're using. Maybe the different ix costs are similar, so this is a good approximation, but I am not familiar. It's probably not possible with the current structure, but are the different ixs something we should consider?

Reply:
Firedancer uses combination of (program_key, first_byte_of_payload) to get finer cost for programs (or ixs?) 

On the other hand, since these costs are to size TXs for block packing, i think stable values are more important than accuracy. But that just one point of view.